corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -129,11 +129,15 @@ hqDefine('cloudcare/js/utils', [
     };
 
     var formplayerLoading = function () {
-        showLoading();
+        if (!sessionStorage.validationInProgress) {
```

Comment:
I'm seeing `validationInProgress` return as a `string` as opposed to a `boolean`, so 'false' is evaluating to `true`. I don't know if this is browser specific (I'm using chrome). Are you seeing something different?

Reply:
[83ff3ba](https://github.com/dimagi/commcare-hq/pull/34035/commits/83ff3ba2f34786fff6ec02c8ce7d7006fffd8efd)